auto_nag/scripts/multifix_regression.py

```diff
@@ -39,6 +42,23 @@ def __merge_comment(tools: ToolsChanges) -> dict:
 
         raise UnexpectedToolsError(list(tools))
 
+    @staticmethod
+    def __merge_keywords(tools: ToolsChanges) -> dict:
+        merged_changes = defaultdict(set)
+        for changes in tools.values():
+            if "keywords" in changes:
```

Comment:
nit: use `not+continue` to avoid to increase the indentation

Reply:
Done in fad48e250a2861e26dc1e4313c3162c6734964b4